paasta_tools/utils.py

```diff
@@ -1132,6 +1132,12 @@ def get_docker_client():
         return Client(base_url=get_docker_host(), **client_opts)
 
 
+def get_running_mesos_docker_containers():
+    client = get_docker_client()
```

Comment:
Nothing else seems to pass in a docker client as an argument anywhere else in utils.py, the one other function that interacts with docker in utils.py creates a client within the function itself (but does not use `get_docker_client`). Happy to move this but I figured I'd follow existing convention even if it's a single convention.